src/contracts/Container/Encore/ConfigurationDumper.php

```diff
@@ -31,7 +31,7 @@ public function __construct(ContainerInterface $containerBuilder)
     }
 
     /**
-     * @param array<string, array<string, array{?'deprecated': bool, ?'alternative': string}>> $webpackConfigNames
+     * @phpstan-param array<string, array<string, array{?'deprecated': bool, ?'alternative': string}>> $webpackConfigNames
```

Comment:
According to https://phpstan.org/writing-php-code/phpdoc-types#array-shapes the question mark should be behind the optional key name:
```suggestion
     * @phpstan-param array<string, array<string, array{'deprecated'?: bool, 'alternative'?: string}>> $webpackConfigNames
```